data/core/game_balance.json

```diff
@@ -355,5 +355,180 @@
     "info": "Sets eternal weather.  Possible values are 'normal', 'clear', 'cloudy', 'light_drizzle', 'drizzle', 'rain', 'rainstorm', 'thunder', 'lightning', 'flurries', 'snowing', 'snowstorm', 'early_portal_storm', 'portal_storm'.  'normal' clears all eternal weather overrides and sets normal weather pattern.  Requires restart of the game after changing value to take effect.",
     "stype": "string_input",
     "value": "normal"
+  },
+  {
+    "type": "EXTERNAL_OPTION",
+    "name": "SPAWN_CHANCE_GUNS",
+    "info": "Set global chance to spawn items from 'guns' category.  Valid values are 0-100.  Set 0 to forbid spawning items from this category.  Default: 100.",
+    "stype": "int",
+    "value": 100
```

Comment:
I believe `spawn_chance` would be better to be placed in https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/item_category.json - that way we won't have to deal with hardcode when we introduce new category.